example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -17,6 +17,7 @@
 		97C146FC1CF9000F007C117D /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 97C146FA1CF9000F007C117D /* Main.storyboard */; };
 		97C146FE1CF9000F007C117D /* Assets.xcassets in Resources */ = {isa = PBXBuildFile; fileRef = 97C146FD1CF9000F007C117D /* Assets.xcassets */; };
```

Comment:
Should this be added to gitingore?

Reply:
Actually not! This looks auto-generated, but there are some fields we might want to manually edit.
https://github.com/flutter/flutter/issues/12639#issuecomment-338037309